oracle/oracle.go

```diff
@@ -0,0 +1,30 @@
+package oracle
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	// TokenIDLength is the length of an NFT token ID
+	TokenIDLength = 32
+)
+
+type updateNFTOracleRequest struct {
+	TokenID [TokenIDLength]byte
+	// this is the unique identifier of the NFT Oracle contract
+	OracleFingerprint []byte
+	Result            []byte
+	OracleAddress     common.Address
```

Comment:
We just need the Attribute Key and Oracle address from the API. Node will take the attribute result and push it to Oracle along with fingerprint